test/acceptance/features/steps/steps.py

```diff
@@ -394,3 +397,40 @@ def etc_cluster_is_running(context, etcd_name):
         print("etcd cluster not present, creating etcd cluster")
         assert etcd_cluster.create() is True, "etcd cluster is not created"
         assert etcd_cluster.is_present() is True, "etcd cluster is not present"
+
+
+@when(u'Invalid Service Binding is applied')
+def invalid_sbr_is_applied(context):
+    sbr = ServiceBinding()
+    # Get resource version of sbr if sbr is available
+    if "sbr_name" in context:
+        json_path = "{.metadata.resourceVersion}"
+        rv = sbr.get_servicebinding_info_by_jsonpath(context.sbr_name, context.namespace.name, json_path)
+        context.resource_version = rv
+    yaml = context.text
+    output = sbr.attempt_to_create(yaml)
+    context.expected_error = output
+
+
+@then(u'Error message "{err_msg}" is thrown')
+def validate_error(context, err_msg):
+    search = re.search(rf'.*{err_msg}.*', context.expected_error)
+    assert search is not None, "Actual error: {context.expected_error}, Expected error: {err_msg}"
+
+
+@then(u'Service Binding "{sb_name}" is not persistent in the cluster')
+def get_sb(context, sb_name):
```

Comment:
can we update the function name to reflect what is going on? the function is not about getting  service binding - it is about checking if the given SB exists in the cluster.

Reply:
Updated PTAL.